### PR TITLE
CI Static check files and test files

### PR DESCRIPTION
**Pull request by satya-gorli**
What this PR does / why we need it:
The PR contains files related to Static Checks (goMetalint, goFmt,) and testing (Unit testing, integration testing, e2e test) jobs
Which issue this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged): fixes #
CI files
Special notes for your reviewer:
Please review the CI Files
Release note:
None

### CODE REVIEW

File: osds/verify.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+fi
+export PATH=${PATH}:${GOPATH}/bin:${GOPATH}/src/github.com/opensds/opensds/osds/bin
+gometalinter --disable-all --enable=gofmt --enable=misspell --enable=golint --exclude=vendor --exclude=test ./...
+if [ $? != 0 ]; then
+       	echo "Please fix the warnings!"
+	echo "Run ./update-gofmt.sh if any warnings in gofmt"
+       	exit 1
+else
+	echo "Gofmt,misspell,golint checks have been passed"
+fi
```
Comment (wisererik) [inline osds/verify.sh]: didn't see any golink, misspell check in CI log

Comment (satya-gorli) [reply osds/verify.sh]: Fixed & Updated verify.sh

**Comment by leonwanghui** [comment]
Please change the base branch to development

**Comment by satya-gorli** [comment]
Created new PR from Development as base branch with PR no. #745

**Pull request closed without merging**